cli/tests/pcluster/validators/test_all_validators/test_scheduler_plugin_all_validators_are_called/scheduler_plugin_1.yaml

```diff
@@ -0,0 +1,263 @@
+    EphemeralVolume:
+      MountDir: /test
+  Dcv:
+    Enabled: true
+    Port: 8443
+    AllowedIps: 0.0.0.0/0
+  CustomActions:
+    OnNodeStart:
+      Script: https://test.tgz
+      Args:
+        - String
+        - stirng2
+    OnNodeConfigured:
+      Script: https://test.tgz
+      Args:
+        - String
+        - stirng2
+  Iam:
+    InstanceRole: arn:aws:iam::aws:role/CustomHeadNodeRole
+Scheduling:
+  Scheduler: plugin
+  SchedulerSettings:
+    SchedulerDefinition:
+      PluginInterfaceVersion: "1.0"
+      Metadata:
+        Version: 1.0.0
+        Name: my-scheduler
+        Documentation: link
+      Requirements:
+        SupportedDistros:
+          X86: ["ubuntu1804", "centos7"]
+          Arm64: ["ubuntu1804"]
+        SupportedRegions: ["cn-north-1", "us-east-1"]
+        QueueConstraints:
+          MaxCount: 5
+        ComputeResourceConstraints:
+          MaxCount: 3
+        RequiresSudoPrivileges: False
+        SupportsClusterUpdate: True
+        SupportedParallelClusterVersions: ">=3.1.0, <=3.4.2"
+      ClusterInfrastructure:
+        CloudFormation:
+          Template: https://bucket/scheduler_plugin/additional_cluster_infrastructure_no_jinja.cfn.yaml
+          Checksum: b4479e35f4e1f60b680f343b21d9dc30c958a6d239974e96a463b4479e35f4e1
+      PluginResources:
+        ClusterSharedArtifacts:
+          - Source: s3://${Region}-aws-parallelcluster.s3.${Region}.${URLSuffix}/plugins/slurm/v1.0.0/artifacts.tar.gz
+            S3BucketOwner: "012345678910"
+            Checksum: b4479e35f4e1f60b680f343b21d9dc30c958a6d239974e96a463b4479e35f4e0
+          - Source: s3://${Region}-aws-parallelcluster.s3.${Region}.${URLSuffix}/plugins/slurm/v1.0.0/artifacts2.tar.gz
+      Events:
+        HeadInit:
+          ExecuteCommand:
+            Command: env
+        HeadConfigure:
+          ExecuteCommand:
+            Command: artifacts/handlers/head_configure_plugin.sh
+        HeadFinalize:
+          ExecuteCommand:
+            Command: artifacts/handlers/head_finalize.sh
+        ComputeInit:
+          ExecuteCommand:
+            Command: env
+        ComputeConfigure:
+          ExecuteCommand:
+            Command: artifacts/handlers/compute_configure_plugin.sh
+        ComputeFinalize:
+          ExecuteCommand:
+            Command: artifacts/handlers/compute_finalize.sh
+        HeadClusterUpdate:
+          ExecuteCommand:
+            Command: artifacts/handlers/head_cluster_update.sh
+        HeadComputeFleetUpdate:
+          ExecuteCommand:
+            Command: artifacts/handlers/head_computefleet_update.sh
+      Monitoring:
+        Logs:
+          Files:
+            - FilePath: /var/log/slurmctld.log
+              TimestampFormat: "%Y-%m-%dT%H:%M:%S.%f"
+              NodeType: ALL
+              LogStreamName: slurmctld.log
+            - FilePath: /var/log/slurmd.log
+              TimestampFormat: "%Y-%m-%dT%H:%M:%S.%f"
+              NodeType: HEAD
+              LogStreamName: slurmctld.log
```

Comment:
Minor: maybe we want to generalize the names of these logs, since the rest of your SchedulerPlugin parameters are generic (for example, the name `my_scheduler`).

Reply:
These log names are just some example to used for the validators. If people use slurm plugin, they can configure the logstream name from the paths.